packages/story-editor/src/components/floatingMenu/elements/elementAlignment.js

```diff
@@ -23,15 +23,80 @@ import { __ } from '@googleforcreators/i18n';
 /**
  * Internal dependencies
  */
-import { IconButton } from './shared';
+import { useStory } from '../../../app';
+import useAlignment from '../../panels/design/alignment/useAlignment';
+import { IconButton, Separator } from './shared';
 
 function ElementAlignment() {
+  const { selectedElements, updateSelectedElements } = useStory(
+    ({ state: { selectedElements }, actions: { updateSelectedElements } }) => ({
+      selectedElements,
+      updateSelectedElements,
+    })
+  );
+
+  const updateElements = (callback) =>
+    updateSelectedElements({ properties: callback });
+
+  const {
+    isDistributionEnabled,
+    handleAlignLeft,
+    handleAlignCenter,
+    handleAlignRight,
+    handleAlignTop,
+    handleAlignMiddle,
+    handleAlignBottom,
+    handleHorizontalDistribution,
+    handleVerticalDistribution,
+  } = useAlignment({ selectedElements, updateElements });
   return (
-    <IconButton
-      Icon={Icons.AlignLeft}
-      title={__('Align elements', 'web-stories')}
-      onClick={() => {}}
-    />
+    <>
+      <IconButton
+        Icon={Icons.AlignLeft}
```

Comment:
The order of the alignment buttons differs in the design menu differs from the order in the design panel.
Design panel:
<img width="300" alt="Screenshot 2022-03-15 at 11 03 31" src="https://user-images.githubusercontent.com/3294597/158343073-5248a188-1804-4931-9ad1-2182bdb221bd.png">

Design menu:
<img width="394" alt="Screenshot 2022-03-15 at 11 04 04" src="https://user-images.githubusercontent.com/3294597/158343164-f6cec361-0651-40d3-b96a-2559f4e38c10.png">

Maybe a design oversight but probably would make sense for these to match. WDYT?

Reply:
Well, that's how they looked in Figma for the new menu, so just implemented as-is. But yeah, it might confuse users. I'll confirm with UX.